internal/cmd/issue/edit/edit.go

```diff
@@ -246,13 +253,27 @@ func handleUserAssign(project, key, assignee string, client *jira.Client) {
 	}
 }
 
+// hasNoChanges checks if any editable fields were provided in the params.
+func hasNoChanges(params *editParams) bool {
```

Comment:
Shouldn't it be this?


```suggestion
func hasNoChanges(params *editParams) bool {
```

And so, it could become a method


```suggestion
func (params *editParams) hasNoChanges() bool {
```

Reply:
Tried to address this in da24dda.